doc/INSTALL.md

```diff
@@ -22,74 +22,115 @@ For the following Linux distributions and package managers 0.8 packages are avai
-`/etc/apt/sources.list`:
+Or alternatively, you can use (if you do not mind many dependences just to add one line to a config file):
 
+```sh
+# Example for Ubuntu Focal
+sudo apt-get install software-properties-common apt-transport-https
+sudo add-apt-repository "deb https://debs.libelektra.org/focal focal main"
 ```
-deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main
-deb-src [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main
+
+If you would like to use our bleeding-edge builds (master builds), append `-unstable` to `<DISTRIBUTION>`.
+
+The `etc/apt/source.list` entry must look like following: `deb https://debs.libelektra.org/<DISTRIBUTION> <DISTRIBUTION>-unstable main`
+
+E.g. `deb https://debs.libelektra.org/focal focal-unstable main`
+
+> NOTE: for Ubuntu Bionic the yamlcpp plugin is excluded due to missing dependencies and therefore the package `libelektra4-yamlcpp` is not available.
+
+### Fedora
+
+We also provide stable and bleeding-edge packages for Fedora 33.
+
+For the stable repository:
+
+```sh
+wget https://rpms.libelektra.org/fedora-33/libelektra.repo -O libelektra.repo;
+sudo mv libelektra.repo /etc/yum.repos.d/;
+sudo yum update
 ```
 
-Which can also be done using:
+Or alternatively you can use dnf to add this repo:
 
 ```sh
-sudo apt-get install software-properties-common apt-transport-https
-echo "deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main" | sudo tee /etc/apt/sources.list.d/elektra.list
+dnf config-manager --add-repo https://rpms.libelektra.org/fedora-33/libelektra.repo
 ```
 
-Or alternatively, you can use (if you do not mind many dependences just to add one line to a config file):
+For our bleeding-edge builds append `-unstable` to the distribution name:
 
 ```sh
-sudo apt-get install software-properties-common apt-transport-https
-sudo add-apt-repository "deb     [trusted=yes] https://debian-buster-repo.libelektra.org/ buster main"
+wget https://rpms.libelektra.org/fedora-33-unstable/libelektra.repo -O libelektra.repo;
+sudo mv libelektra.repo /etc/yum.repos.d/;
+sudo yum update
 ```
 
-If you want to rebuild Elektra from Debian unstable or
-our repositories, add a `deb-src` entry to `/etc/apt/sources.list`
-and then run:
+Or alternatively you can use dnf to add this repo:
 
 ```sh
-apt-get source -b elektra
+dnf config-manager --add-repo https://rpms.libelektra.org/fedora-33-unstable/libelektra.repo
 ```
 
 ### Install
 
 To get all packaged plugins, bindings and tools install:
 
 ```sh
+# For Debian based distributions
 apt-get install libelektra4-all
+# For Fedora based distributions
+dnf install libelektra4-all
 ```
 
 For a small installation with command-line tools available use:
 
 ```sh
+# For Debian based distributions
 apt-get install elektra-bin
+# For Fedora based distributions
+dnf install elektra-bin
+```
+
+To install all debugsym/debuginfo packages:
```

Comment:
Is this included in `libelektra4-all`?

Reply:
Yes but only as a "Suggests" dependency and not as a "Depends". On RPM this is included as a "Recommends" dependency.

Reply:
Ok, then the text is okay as it is.